articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -15,52 +15,52 @@ ms.date: 10/20/2019
 
 Sign in to the [Azure portal](https://portal.azure.com) with your Azure account credentials.
 
-## Create your logic app
+## Create your Logic App
 
 1. From the Azure home page, in the search box, find and select **Logic Apps**.
 
    ![Find and select "Logic Apps"](./media/quickstart-create-first-logic-app-workflow/find-select-logic-apps.png)
 
 1. On the **Logic Apps** page, select **Add**.
 
-   ![Add new logic app](./media/quickstart-create-first-logic-app-workflow/add-new-logic-app.png)
+   ![Add new Logic App](./media/quickstart-create-first-logic-app-workflow/add-new-logic-app.png)
 
-1. Under **Create logic app**, provide details about your logic app as shown here. After you're done, select **Create**.
+1. Under **Create Logic App**, provide details about your Logic App as shown below. After you are done, select **Create**.
 
-   ![Provide details for new logic app](./media/quickstart-create-first-logic-app-workflow/create-logic-app-settings.png)
+   ![Provide details for new Logic App](./media/quickstart-create-first-logic-app-workflow/create-logic-app-settings.png)
 
    | Property | Value | Description |
    |----------|-------|-------------|
-   | **Name** | <*logic-app-name*> | Your logic app name, which can contain only letters, numbers, hyphens (`-`), underscores (`_`), parentheses (`(`, `)`), and periods (`.`). This example uses "My-First-Logic-App". |
+   | **Name** | <*logic-app-name*> | Your Logic App name, which can contain only letters, numbers, hyphens (`-`), underscores (`_`), parentheses (`(`, `)`), and periods (`.`). This example uses "My-First-Logic-App". |
    | **Subscription** | <*Azure-subscription-name*> | Your Azure subscription name |
    | **Resource group** | <*Azure-resource-group-name*> | The name for the [Azure resource group](../azure-resource-manager/resource-group-overview.md) used to organize related resources. This example uses "My-First-LA-RG". |
-   | **Location** | <*Azure-region*> | The region where to store your logic app information. This example uses "West US". |
+   | **Location** | <*Azure-region*> | The region where to store your Logic App information. This example uses "West US". |
    | **Log Analytics** | Off | Keep the **Off** setting for diagnostic logging. |
    ||||
 
-1. After Azure deploys your app, on the Azure toolbar, select **Notifications** > **Go to resource** for your deployed logic app.
+1. After Azure deploys your app, on the Azure toolbar, select **Notifications** > **Go to resource** for your deployed Logic App.
 
-   ![Go to newly created logic app resource](./media/quickstart-create-first-logic-app-workflow/go-to-new-logic-app-resource.png)
+   ![Go to newly created Logic App resource](./media/quickstart-create-first-logic-app-workflow/go-to-new-logic-app-resource.png)
 
-   Or, you can find and select your logic app by typing the name in the search box.
+   Alternatively, you can find and select your Logic App by typing the name in the search box.
```

Comment:
- "Or" is OK to promote friendlier tone, please revert.

Reply:
Currently, you're reviewing an outdated file

Reply:
already been reverted